utils/tests_fetcher.py

```diff
@@ -1054,7 +1054,7 @@ def infer_tests_to_run(
         # Then we grab the corresponding test files.
         test_map = create_module_to_test_map(reverse_map=reverse_map, filter_models=filter_models)
         for f in modified_files + impacted_files:
-            if f in test_map:
+            if f in test_map and test_map[f] is not None:
```

Comment:
`test_map[f]` seems to be empty in some situations, which breaks the line below.

More specifically, in the commit before this change (`f4e04b7f519cf8c79b80233cd7dbdd17e9586038`), this was the CI for the test fetcher: https://app.circleci.com/pipelines/github/huggingface/transformers/102539/workflows/ebaf3657-9fab-4718-8240-ad1f350eb9cf/jobs/1365693